src/include/processor/operator/persistent/copy_to_csv.h

```diff
@@ -26,22 +26,24 @@ struct CopyToCSVInfo : public CopyToInfo {
     }
 };
 
-class CopyToCSVLocalState : public CopyToLocalState {
+class CopyToCSVLocalState final : public CopyToLocalState {
 public:
-    void init(CopyToInfo* info, storage::MemoryManager* mm, ResultSet* resultSet) final;
+    void init(CopyToInfo* info, storage::MemoryManager* mm, ResultSet* resultSet) override;
```

Comment:
if you mark class as final, then you can remove all override. Apply across this PR.